cmsimple/classes/Page.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace XH;
+
+/**
+ * Initiates a page object. Its methods act as a wrapper
+ * of the XH/Pages methods for better OOP access
+ *
+ * @category CMSimple_XH
+ * @package  XH
+ * @author   The CMSimple_XH developers <devs@cmsimple-xh.org>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.en.html GNU GPLv3
+ * @link     http://cmsimple-xh.org/
+ * @since    1.8
+ */
+
+Class Page
+{
+    
+    /**
+     * Instance of XH/Pages.
+     *
+     * @var pages
```

Comment:
The `@var` tag should also specify the type, so instead of `pages` it should be `XH\Pages` (see above). Same issue below.